### PR TITLE
1995-STEM: "Based on you responses" should be "Based on your responses"

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
A typographical error is present on the "Rogers STEM Scholarship eligibility" page of the 1995-STEM form. "You" in the header of the box indicating why a user is not eligible should be "your". See screenshot below

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/6393

## Testing done
Local testing 
## Screenshots

Screen Shot 2020-03-06 at 7 53 10 AM

## Acceptance criteria
- [x] Text reads `your` and not `you`

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx [hunk 1]
```diff
@@ -140,6 +140,11 @@ export class StemEligibilityView extends React.Component {
       <div className={divClassName}>
         <fieldset className="schemaform-field-template schemaform-first-field">
           <legend className={legendClassName}>
+            <div className="sr-only">
+              {this.renderIsEdithNourseRogersScholarshipCheck()}
```
Comment (zurbergram) [inline src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx]: Each of these methods render a `li` element they should be wrapped in a `ul`

Comment (ksantiagomorgan6) [reply src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx]: added <ul>

**Review by zurbergram — APPROVED** [review]

**Review by sonntag-adam — APPROVED** [review]

File: src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx [hunk 2]
```diff
@@ -140,6 +140,13 @@ export class StemEligibilityView extends React.Component {
       <div className={divClassName}>
         <fieldset className="schemaform-field-template schemaform-first-field">
           <legend className={legendClassName}>
+            <div className="sr-only">
```
Comment (1Copenut) [inline src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx]: While reviewing this code a second time on `localhost` it feels too repetitive to include the list of responses a second time. Let's just remove this block `<div className="sr-only">` and the list inside it. No additions will be necessary.

**Review by 1Copenut — CHANGES_REQUESTED** [review]

File: src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx (hunk 2 above)
Comment (ksantiagomorgan6) [reply src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx]: Code removed

**Review by 1Copenut — APPROVED** [review]

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**